lib/auth/auth.go

```diff
@@ -390,6 +398,13 @@ func NewServer(cfg *InitConfig, opts ...ServerOption) (*Server, error) {
 		)
 	}
 
+	go func() {
```

Comment:
This goroutine will never terminate. We should select on `process.ExitContext()`.

I think we should move this under `runPeriodicOperations()` in this same file.